packages/quantic/force-app/main/default/lwc/quanticSort/quanticSort.js

```diff
@@ -38,8 +44,13 @@ import {LightningElement, track, api} from 'lwc';
       criterion={sortOptionCriterion}
     ></c-quantic-sort-option></c-quantic-sort>
  */
-
 export default class QuanticSort extends LightningElement {
+  /**
+   * The sort variant. Accepted variants include `default` and `wide`.
+   * @api
+   * @type {'default'|'wide'}
+   */
+  @api variant = 'default';
```

Comment:
Can you do this instead?

```suggestion
  @api variant = SORT_VARIANTS.DEFAULT;
```

Reply:
nah I tried, its giving me a type error:
<img width="520" alt="image" src="https://github.com/user-attachments/assets/f5a7cbeb-f4c4-47d5-9c17-d8614ccacc96">
either I set the @type to string, or leave it as is and set the default value to 'default'